frontend/src/App.test.jsx

```diff
@@ -0,0 +1,80 @@
+import React from "react"
+import { render, screen, waitFor } from '@testing-library/react'
```

Comment:
This packege isn't in package.json. we need to make it work with package.json in the root folder. lets do a screen share at some point to fix this